numpy/polynomial/polynomial.py

```diff
@@ -1443,6 +1443,10 @@ class Polynomial(ABCPolyBase):
     '+', '-', '*', '//', '%', 'divmod', '**', and '()' as well as the
     attributes and methods listed in the `ABCPolyBase` documentation.
 
+    Please note that ``Polynomial.coef`` are coefficients in the scaled
```

Comment:
Can we put this in the docs for `coef` instead?

Reply:
I don't think that's going to work. In IPython:

```
[ins] In [18]: p = np.polynomial.Polynomial.fit(a, b, 1)                                                                                                                                                              
Out[18]: Polynomial([3.5, 0.5], domain=[0., 1.], window=[-1.,  1.])

[ins] In [20]: p.coef?                                                                                                                                                                                            
# docs for ndarray
[ins] In [21]: help(p.coef)
# likewise                            
```

Reply:
If nothing else, this docstring belongs in `ABCPolyBase` not here - it applies to all subclasses.

Reply:
Sure, but when I, the Well Intentioned User intending to Write Code According To New Standards, look for documentation on this, I'm going to find [docs for Polynomial](https://docs.scipy.org/doc/numpy/reference/generated/numpy.polynomial.polynomial.Polynomial.html#numpy.polynomial.polynomial.Polynomial), not for `ABCPolyBase`; and given that `ABCPolyBase` is not actually linked from `Polynomial`, the odds of me going looking for them fall off pretty sharply.

I'd keep it here and add it to ABCPolyBase, if you don't mind.

A separate issue is that I only found that page after correcting for the fact that google prioritizes old numpy doc versions for some reason.

Reply:
Does the link on the line immediately above not work?

Reply:
No, actually, if you take a look at the rendered docs I linked, it's not a hyperlink - it's formatted in bold, though.

Reply:
No, actually, if you take a look at the rendered docs I linked, it's not a hyperlink - it's formatted in bold, though.

Reply:
The documentation for `p` is inherited from `ABCPolyBase`.

Reply:
I tried it again:
```
>>>help(p)
Help on Polynomial in module numpy.polynomial.polynomial object:

class Polynomial(numpy.polynomial._polybase.ABCPolyBase)
 |  Polynomial(coef, domain=None, window=None)
 |  
 |  A power series class.
 |  
 |  The Polynomial class provides the standard Python numerical methods
 |  '+', '-', '*', '//', '%', 'divmod', '**', and '()' as well as the
 |  attributes and methods listed in the `ABCPolyBase` documentation.
 |  
 |  Parameters
 |  ----------
 |  coef : array_like
 |      Polynomial coefficients in order of increasing degree, i.e.,
 |      ``(1, 2, 3)`` give ``1 + 2*x + 3*x**2``.

```

I may be misunderstanding something, but if it were to be inheriting or copying ABCPolyBase's docstring, I'd expect to see the "attributes and methods listed in the `ABCPolyBase` documentation." line as seen in the change above.